nilearn/decomposition/tests/test_canica.py

```diff
@@ -1,11 +1,16 @@
 """Test CanICA"""
+import os
```

Comment:
import probably not needed anymore if we give up on skipping the test for Neurodebian